src/main/com/fulcrologic/statecharts/visualization/visualizer.cljs

```diff
@@ -161,33 +184,39 @@
                       :height           (str (get-in (or node-id->layout node-id->size) [:ROOT :height]) "px")
                       :top              (get-in (or node-id->layout node-id->position) [:ROOT :y] 0)
                       :left             (get-in (or node-id->layout node-id->position) [:ROOT :x] 0)
-                      :background-color "white"}}
-      (elk/render-edges layout)
+                      :backgroundColor "white"}}
+      (when layout (elk/render-edges layout))
       (mapv
         (fn [{:keys [id initial? compound? node-type children] :as node}]
           (if initial?
-            (dom/div {:key   (pr-str id)
+            (dom/div {:id   (pr-str id)
```

Comment:
Adding an id is a bad idea. Two charts might have the same node IDs, so this would cause DOM-level id conflicts.

Reply:
True. Leftover from visual debugging.